test/index.test.js

```diff
@@ -8,97 +8,92 @@ const {
+    });
+  });
 
   it('should handle customNetwork url correctly', function () {
-    const result = createCustomAccountLink('foo', 'https://data-seed-prebsc-1-s1.binance.org:8545')
-    assert.strictEqual(result, 'https://data-seed-prebsc-1-s1.binance.org:8545/address/foo', 'should return binance testnet address url')
-  })
-})
+    const result = createCustomAccountLink('foo', 'https://data-seed-prebsc-1-s1.binance.org:8545');
+    assert.strictEqual(result, 'https://data-seed-prebsc-1-s1.binance.org:8545/address/foo', 'should return binance testnet address url');
+  });
+});
 
 // `https://${prefix}etherscan.io/tx/${hash}`
 describe('explorer-link', function () {
   describe('by networkId', function () {
     it('should handle mainnet correctly', function () {
-      const result = createExplorerLink('foo', '1')
-      assert.strictEqual(result, 'https://etherscan.io/tx/foo', 'should handle mainnet')
-    })
-  
+      const result = createExplorerLink('foo', '1');
+      assert.strictEqual(result, 'https://etherscan.io/tx/foo', 'should handle mainnet');
+    });
+
     it('should handle ropsten correctly', function () {
-      const result = createExplorerLink('foo', '3')
-      assert.strictEqual(result, 'https://ropsten.etherscan.io/tx/foo', 'should handle ropsten')
-    })
-  
+      const result = createExplorerLink('foo', '3');
+      assert.strictEqual(result, 'https://ropsten.etherscan.io/tx/foo', 'should handle ropsten');
+    });
+
     it('should have null as a prefix', function () {
-      const result = createExplorerLink('foo', '10')
-      assert.strictEqual(result, '', 'should return an empty string')
-    })
-  })
+      const result = createExplorerLink('foo', '10');
+      assert.strictEqual(result, '', 'should return an empty string');
+    });
+  });
 
-  describe('by networkId', function () {
+  describe('by chainId', function () {
     it('should handle mainnet correctly', function () {
-      const result = createExplorerLinkForChain('foo', '0x1')
-      assert.strictEqual(result, 'https://etherscan.io/tx/foo', 'should handle mainnet')
-    })
-  
+      const result = createExplorerLinkForChain('foo', '0x1');
+      assert.strictEqual(result, 'https://etherscan.io/tx/foo', 'should handle mainnet');
+    });
+
     it('should handle ropsten correctly', function () {
-      const result = createExplorerLinkForChain('foo', '0x3')
-      assert.strictEqual(result, 'https://ropsten.etherscan.io/tx/foo', 'should handle ropsten')
-    })
-  
+      const result = createExplorerLinkForChain('foo', '0x3');
+      assert.strictEqual(result, 'https://ropsten.etherscan.io/tx/foo', 'should handle ropsten');
+    });
+
     it('should have null as a prefix', function () {
-      const result = createExplorerLinkForChain('foo', '0xa')
-      assert.strictEqual(result, '', 'should return an empty string')
-    })
-  })
-  
-  it('should handle customNetwork url correctly', function () {
```

Comment:
This test was removed - it was flagged for having a duplicate description. It was a dupe of the earlier test `'should handle ropsten correctly'`, under `by chainId`.